app/resources/admin.html.template

```diff
@@ -83,45 +83,74 @@
         return false;
       }
       used_languages.push(language_code);
-      languages_to_titles[language_code] = language_title;
     }
 
     // Copy the dropdowns and the titles into the text boxes that will
     // actually be POSTed (language_menu_options and repo_titles).
     $('language_menu_options').value = JSON.stringify(used_languages);
-    $('repo_titles').value = JSON.stringify(languages_to_titles);
-
+    var custom_message_table_ids = [
+      'repo_titles', 
+      'start_page_custom_htmls', 
+      'results_page_custom_htmls', 
+      'view_page_custom_htmls', 
+      'seek_query_form_custom_htmls', 
+      'footer_custom_htmls'
+    ];
+    for(var id_num = 0; id_num < custom_message_table_ids.length; id_num++){
+      var table_name = custom_message_table_ids[id_num];
+      for(var i = 1; i < dropdown_index; i++){
```

Comment:
Shouldn't we start with i=0? Like

for(var i = 0; i < used_languages.length; i++){ ...

I may be missing something.